test/ejabberd_tests/tests/ejabberdctl_SUITE.erl

```diff
@@ -595,10 +608,175 @@ stats_host(Config) ->
                 {"0\n", 0} = ejabberdctl("stats_host", ["onlineusers", SecDomain], Config)
         end).
 
+
+
+%%-----------------------------------------------------------------
+%% Improve coverage
+%%-----------------------------------------------------------------
+
+
+
+simple_register(Config) ->
+    %% given
+    Domain = escalus_ct:get_config(ejabberd_domain),
+    {Name, Password} = {<<"tyler">>, <<"durden">>},
+    %% when
+    {_, _} = ejabberdctl("register", [Name, Domain, Password], Config),
+    {R2, _} = ejabberdctl("registered_users", [Domain], Config),
+    %% then
+    {match, _} = re:run(R2, ".*(" ++binary_to_list(Name)++").*").
+
+simple_unregister(Config) ->
+    %% given
+    Domain = escalus_ct:get_config(ejabberd_domain),
+    {Name, _} = {<<"tyler">>, <<"durden">>},
+    %% when
+    {_, _} = ejabberdctl("unregister", [Name, Domain], Config),
+    {R2, _} = ejabberdctl("registered_users", [Domain], Config),
+    %% then
+    nomatch = re:run(R2, ".*(" ++binary_to_list(Name)++").*").
+
+register_twice(Config) ->
+    %% given
+    Domain = escalus_ct:get_config(ejabberd_domain),
+    {Name,  Password} = {<<"tyler">>, <<"durden">>},
+    %% when
+    {_, _} = ejabberdctl("register", [Name, Domain, Password], Config),
+    {R, _} = ejabberdctl("register", [Name, Domain, Password], Config),
+    %% then
+    {match, _} = re:run(R, ".*(already registered).*"),
+    {_, _} = ejabberdctl("unregister", [Name, Domain], Config).
+
+
+
+backup_restore_mnesia(Config) ->
+    %% given
+    TableName = passwd,
+    TableSize = rpc_call(mnesia, table_info, [TableName, size]),
+    io:format("Table size is ~n~p~n", [TableSize]),
+    %% Table passwd should not be empty
```

Comment:
In fact this table will be empty in all jobs on travis when ODBC or LDAP is used for user storage but I think we can live with this.